src/components/VideoForm/VideoForm.jsx

```diff
@@ -0,0 +1,98 @@
+import { ErrorMessage } from '@hookform/error-message';
+import { useForm } from 'react-hook-form';
+
+import useStore from '../../hooks/useStore';
+import StyledButton from '../StyledButton';
+import StyledForm from '../StyledForm';
+import StyledInput from '../StyledInput';
+import StyledInputWarning from '../StyledInputWarning';
+import StyledLable from '../StyledLabel';
+
+export default function VideoForm() {
+	const setVideos = useStore(state => state.setVideos);
+	const videos = useStore(state => state.videos);
+
+	const {
+		register,
+		handleSubmit,
+		resetField,
+		formState: { errors },
+	} = useForm({ criteriaMode: 'all' });
+
+	function onSubmit(data) {
+		setVideos(data);
+		resetField('YouTubeLink');
+		resetField('videoTitle');
```

Comment:
It should be possible to reset both fields at the same time:

```suggestion
		reset();
```